astroscaler/groups.py

```diff
@@ -122,7 +122,11 @@ def is_cooling_down(self, cooldown):
                 return True
             cooldown_limit = datetime.now(endtime.tzinfo) - timedelta(seconds=cooldown)
             if endtime > cooldown_limit:
```

Comment:
what is `endtime`? is that when the cooldown started? yeah it is... confusing. Can I request that the var name be renamed to `cooldown_starttime` or `policy_starttime`? i don't know what `endtime` is suppose to mean but i thought it meant when the cooldown was suppose to end. very confusing.